UIKitTests/Gestures/TouchHandlingTests.swift

```diff
@@ -80,9 +82,15 @@ class TouchHandlingTests: XCTestCase {
 private extension TouchHandlingTests {
     class ResponderView: UIView {
         var touchesMovedWasCalled = false
+        var touchesEndedWasCalled = false
+
         override func touchesMoved(_ touches: Set<UITouch>, with event: UIEvent?) {
             touchesMovedWasCalled = true
         }
+
+        override func touchesEnded(_ touches: Set<UITouch>, with event: UIEvent?) {
```

Comment:
Hey @janek, cool that you've expanded this test.

It would be great if you could add a test for the two boolean values (i.e. when `cancelsTouchesInView` is `true` – the default – but also when it's `false`)

Reply:
thanks! 

that's actually there, we had preexisting `testCancelsTouchesInView()` and `testDoesNotCancelTouchesInView()` (above). They were both only testing if `touchesMoved` was called (or not). My change is to extend that to test for `touchesEnded` aswell.

I was only asking myself if maybe there is another `touchesSomething` to test for, but I looked now and there's only `Began`. I assume it should be always called, regardless of the `cancelsTouchesInView` value. I'm happy to add it there for completeness if that makes sense!

Reply:
From memory, `touchesBegan` works slightly differently on iOS. If the gesture recogniser fails or doesn't exist then (and only then) will `touchesBegan` be called. So I don't think we can test this in the same way. If you like you can make an issue for that but let's not hold up this bug fix for that reason

Reply:
Ah ok sorry I missed the `XCAssert*False*` in the initial case. Cool then this looks fine to me! Would be great if you could make an issue for the other bit, I'm pretty sure we're not doing it 100% "correctly" right now though

Reply:
cool, I'm happy to!